app/ui/timelinepage.ui

```diff
@@ -295,6 +295,72 @@
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="LayerVisibilityBox">
+         <property name="title">
+          <string>Layer Visibility</string>
+         </property>
+         <layout class="QVBoxLayout" name="LayerVisibilityLayout">
+          <item>
+           <widget class="QLabel" name="LabelVisibilityText">
+            <property name="text">
+             <string>Startup option</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QComboBox" name="layerVisibilityComboBox">
+            <item>
+             <property name="text">
+              <string>Current layer only</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>Relative</string>
+             </property>
+            </item>
+            <item>
+             <property name="text">
+              <string>All Layers</string>
+             </property>
+            </item>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLabel" name="labelVisibility">
+            <property name="text">
+             <string>When layer visibility is relative (gray dot)</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <layout class="QHBoxLayout" name="horizontalLayout_2">
+            <property name="topMargin">
+             <number>0</number>
+            </property>
+            <item>
+             <widget class="QSlider" name="visibilitySlider">
+              <property name="maximum">
+               <number>100</number>
+              </property>
+              <property name="orientation">
+               <enum>Qt::Horizontal</enum>
+              </property>
+             </widget>
+            </item>
+            <item>
+             <widget class="QSpinBox" name="visibilitySpinbox">
```

Comment:
I've fixed it now, looks like some rounding error but I don't understand how that would stop the stepper...

Additionally that calculation was unnecessary since the value was already correct